public/app/plugins/panel/timeseries/plugins/ContextMenuPlugin.tsx

```diff
@@ -42,7 +42,7 @@ export const ContextMenuPlugin: React.FC<ContextMenuPluginProps> = ({
 
   const onClick = useCallback(() => {
     setIsOpen(!isOpen);
-  }, [setIsOpen]);
+  }, [isOpen]);
```

Comment:
Question: what is best practice here? Use a dependency on the `set-function` or the value?

Reply:
value, always. the set function shouldn't be a dependency.